compliance_checker/imos/imos.py

```diff
@@ -369,31 +366,68 @@ def check_time_coverage(self, dataset):
                                 BaseCheck.HIGH)
 
         if result.value:
+            date_attribute_format = '%Y-%m-%dT%H:%M:%SZ'
+
+            time_var = dataset.dataset.variables.get('TIME', None)
+            time_min = np.amin(time_var.__array__())
+            time_max = np.amax(time_var.__array__())
+
+            time_units = getattr(time_var, "units", None)
+            time_calendar = getattr(time_var, "calendar", None)
```

Comment:
This leads to an exception when calendar attribute is missing. Let's make the default value "gregorian" instead of None.